.env

```
@@ -28,6 +23,8 @@ HASH_KRATOS_PG_DATABASE=kratos
 HASH_GRAPH_PG_USER=graph
 HASH_GRAPH_PG_PASSWORD=graph
 HASH_GRAPH_PG_DATABASE=graph
+HASH_GRAPH_API_HOST=localhost
+HASH_GRAPH_API_PORT=4000
```

Nit: whitespace between the `.env` variables dedicated to specific services
```suggestion
HASH_GRAPH_PG_DATABASE=graph

HASH_GRAPH_API_HOST=localhost
HASH_GRAPH_API_PORT=4000
```